### ISSUE TITLE
Read through HN post to collect suggestions

### ISSUE THREAD

**Issue by almarklein** [almarklein/timetagger#16]
https://news.ycombinator.com/item?id=25762715#25765847

## Suggestions

* [x] Possibility to delete a record.
* [x] The icon in the tab is hard to see on a dark background.
* [x] The "House" for "Snap to now" and calendar for time-range feel unintuitive, but that is subjective. 
* [x] That burger menu is, pardon me, somewhat of a problem. There is so much information. I can see "Install App" in the installed App. "Export" happens there but also on the main screen. Which features belong where? Im missing a hierarchy here.
* [x] Your hint text in the "Manage tags" dialog is too long. 
* [x] Is that a refresh button next to the burger menu (left side)? I can click an empty button. Wait, what?
* [x] I really do like that infinite scroll and stuff. Looks cool. On the other hand it is quite easy to get lost.
* [x] Please don't require headings over button groups at all (and then even mix that with my initials!) 
* [x]  the dashed line indicating 'now' is really easy to overlook and it should be visually more prominent (other color, thicker, ...)
* [x] In the overview (this week, this month, ...) the records should be display somehow. Indicating records visually in the overviews makes it easier to grasp the work distribution over longer periods of time. Additionally, if I mess up the month/year of a record (which I frequently do ;-) ) I have no chance to find it again except that I scroll through every day and look for it. So maybe assigning a separate color to each tag and display them in the big grey fields next to the time line would work... 
* [x] Checkout ActivityWatch
* [x] Add to https://wintermute.app/repositories
* [x] Some people seem to be confused about the representation of records in the timeline.
* [x] Many people seem to like the interactive timeline. (One post seemed both.)
* [x] Faster way to start a timer, e.g. via a keyboard shortcut.
* [x] Combine the two ways to start a timer into one button in *some smart way*. Perhaps in the start-dialog you can also stop it directly?
* [x] When selecting tags on the right, also hide (or gray out) records on the left.
* [x] In the save to csv, include a column for the just the tags
* [ ] Make sure you have a robust plugin layer as people will want to integrate this into their existing systems.
* [ ] Public API so other tools can hook into it.
* [x] Allow a company to purchase N licenses at once.
* [x] Does not work on Safari mobile iOS 14.3.
* [x] I would love some transitions on the popup elements
* [x] Keyboard control over the forms :)
* [x] I asked whether a CLI interface would be of interest. 5 ppl showed interest.
* [x] via email: use a CLA to resolve copyright vs duallicensing issue.
* [x] Explain the low price (no free users).

## Things people were excited about

* I think the fact being open source is the main source for the popularity of that post.
  * Able to see the code.
  * In case I like the service and it is discontinued for any reason, able to host it by myself. 
  * Adds another layer of trust and I have the feeling I can influence the product.
* That was perhaps the most compelling landing page I've ever seen. When you mentioned the lack of cookie banner I found it strangely exciting.
* I really like the concept of the continuous scrolling main time bar on the left.
* Also one toxic comment without further explanation: "The UI makes me want to vomit."

**Comment by peppobon** [comment]
It would be useful for todo list and similar to add a 'Start later' option under Start record/Time.

**Comment by almarklein** [comment]
Thanks for the suggestion @peppobon! I think we could change the "started earlier" to allow a time in the future (and perhaps change the option's name).

**Comment by peppobon** [comment]
Thanks @almarklein, it would be perfect.
Here is another one: it would be great to have a text field in the Edit record panel to attach notes/comments

**Comment by almarklein** [comment]
@peppobon I've looked into setting a start time in the future, but it will make the UX very complex. E.g. a record will be started, but is not actually running, making it hard to find back, etc.  So I'm deciding against this approach.  Perhaps you can explain your use-case/work-flow a bit more - there might be other solutions.

**Comment by peppobon** [comment]
Thanks Almar for looking into this, appreciated.
You are right, the UX gets too complex. I realized I can start a record at
the current timestamp and use a #todo tag to mark it appropriately.
This would do what I need keeping things simple.

By the way, great app; I discovered it a few days ago and I liked it at
first sight. I am starting using it instead of trello/timecamp.
Will support you with a subscription.
